A3-channel-tracing.md

```diff
@@ -21,48 +21,35 @@ This design uses the same terminology discussed in the [channelz](A14-channelz.m
 In addition, the following terminology will be used:
 
 1.  a "Trace event" is an interesting thing that happens to a channel. Examples include things like creation, address resolution, subchannel creation, connectivity state changes.
-2. "Trace data" is all of the data that is contained for one channel. This includes a list of _Trace events_, as well as metadata like the timestamp at which the channel was created.
-3.  a "Tracing object" is an in-memory data structure responsible for holding the _Trace data_ for a single channel.
+2.  a "Channel trace" is a data structure responsible for holding all data for a single channel. This includes a list of _Trace events_, as well as metadata like the timestamp at which the channel was created. The _Channel trace_ also holds a list of _Channel trace_ object of its child channels (if it has any).
```

Comment:
The last sentence here is confusing, for two reasons.

First, I don't think you mean a list of objects; I think you mean a list of _pointers_ to objects.

Second, I thought the channel trace was going to contain references to the channelz data of its child channels, not to the channel trace objects in those channelz objects.

Reply:
Sorry, this was another case of a leftover from writing the initial draft of this design before the channelz design was written. I have tried to be more careful, in this round of comments, of making sure the design fits well with channelz.

The Channel trace will hold a list of _references_ to the channels or subchannel mentioned in its list of trace events.